torch/_torch_docs.py

```diff
@@ -8174,6 +8174,54 @@ def merge_dicts(*dicts):
 """,
 )
 
+add_docstr(
+    torch.narrow_copy,
+    r"""
+narrow_copy(input, dim, start, length, *, out=None) -> Tensor
+
+Same as :meth:`Tensor.narrow` except returning a copy rather
+than shared storage.  This is primarily for sparse tensors, which
+do not have a shared-storage narrow method.  Calling ``narrow_copy``
+with ``dimemsion > self.sparse_dim()`` will return a copy with the
```

Comment:
This restriction does not actually in the example below you narrow with `dimension=0` and `self.sparse_dim() = 2`.

Reply:
I'm not sure what this means, This was moved from `torch/_tensor_docs.py` can you propose a different wording?